sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StorageHelperTests.cs

```diff
@@ -15,13 +15,41 @@ public class StorageHelperTests
         private static readonly char ds = Path.DirectorySeparatorChar;
 
         [Fact]
-        public void VerifyConfiguredDirectory()
+        public void VerifyGetStorageDirectory_Configured()
         {
             var directoryPath = StorageHelper.GetStorageDirectory(
-                platform: new MockPlatform(),
-                configuredStorageDirectory: $"C:{ds}Temp");
+                platform: new MockPlatform
+                {
+                    UserName = "ApplicationPoolIdentity",
+                    ProcessName ="w3wp",
+                    ApplicationBaseDirectory = "C:\\inetpub\\wwwroot",
+                },
+                configuredStorageDirectory: $"C:{ds}Temp",
```

Comment:
Is this a common practice?

Reply:
No, this was an ugly hack to allow these tests to run on both Windows and Linux.
Internally, `Path.Combine` will use either `\` or `/` depending on the OS running the unit tests.

I can investigate some alternatives.

Reply:
Updated.